.env.example

```
@@ -0,0 +1,49 @@
+# GitHub App Configuration (Required for webhook processing)
+# -----------------------------------------------------------------------------
+GITHUB_APP_ID=your_github_app_id
+GITHUB_PRIVATE_KEY_PATH=/path/to/github_private_key.pem
+GITHUB_WEBHOOK_SECRET=your_webhook_secret
+
+# -----------------------------------------------------------------------------
+# OpenAI API Configuration (Required for AI-powered PR reviews)
+# -----------------------------------------------------------------------------
+OPENAI_API_KEY=your_openai_api_key
+OPENAI_MODEL=gpt-4o-mini
+OPENAI_MAX_TOKENS=1000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENAI_MAX_TOKENS key should go before the OPENAI_MODEL key

(UnorderedKey)